src/runtimes/node/parser/exports.ts

```diff
@@ -59,12 +64,6 @@ export const traverseNodes = (nodes: Statement[], getAllBindings: BindingMethod)
       return
     }
 
-    if (isESMDefaultExport(node)) {
-      hasDefaultExport = true
-
-      return
-    }
-
```

Comment:
Moving later and not returning because the node might be both a config export _and_ a default export.